docker/Dockerfile

```diff
@@ -1,88 +1,71 @@
-FROM ruby:3.1.3-alpine3.17
-MAINTAINER Ministry of Justice, Claim for crown court defence <crowncourtdefence@digital.justice.gov.uk>
+FROM ruby:3.1.3-alpine3.17 AS base
+
+# Stage 1: installs gems and npm packages
+FROM base AS builder
 
-# fail early and print all commands
 RUN set -ex
 
-# build dependencies:
-# -virtual: create virtual package for later deletion
-# - build-base for alpine fundamentals
-# - ruby-dev/libc-dev for compiling raindrops, at least
-# - libxml2-dev/libxslt-dev for nokogiri, at least
-# - postgres-dev for pg/activerecord gems
-# - git for installing gems referred to use as git:// uri
-# - yarn for js dependency management
-#
-# runtime dependencies:
-# - file: for paperclip file type spoofing check
-# - nodejs: for ExecJS and asset compilation
-# - runit for process management (because we have multiple services)
-# - libreoffice: for pdf conversion
-# - ttf-freefont: needed for libreoffice
-# - redis: for backend key-value store
-# - postgresql-client - only needed for database dump
-#
-RUN apk --no-cache add --virtual build-dependencies \
-                    build-base \
-                    libxml2-dev \
-                    libxslt-dev \
-                    postgresql-dev \
-                    git \
-                    yarn \
-                    gmp=6.2.1-r2 \
-&& apk --no-cache add \
-                  file \
-                  nodejs \
-                  linux-headers \
-                  runit \
-                  ttf-freefont \
-                  libreoffice \
-                  redis \
-                  postgresql-client
-
-# add non-root user and group with alpine first available uid, 1000
-RUN addgroup -g 1000 -S appgroup \
-&& adduser -u 1000 -S appuser -G appgroup
+ENV APP_HOME /app
```

Comment:
Not especially. Stage 2 has to use `/usr/src/app` as the working directory, otherwise subsequent deploy steps fail. I think I was just trying to simplify the first stage, but it probably makes sense for them to be consistent. I'll change this one.